Code/GraphMol/catch_graphmol.cpp

```diff
@@ -3001,3 +3001,180 @@ TEST_CASE("github #4642: Enhanced Stereo is lost when using GetMolFrags") {
           Atom::ChiralType::CHI_UNSPECIFIED);
   }
 }
+
+TEST_CASE("Remove atom updates bond ENDPTS prop") {
```

Comment:
Not necessary, but I don't think it would be a terrible idea to have a new test file for these: `catch_organometallics.cpp`